extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -43,11 +46,20 @@ public static function onMakeGlobalVariablesScript( &$vars ) {
 	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
 		$helper = new Helper();
 
+		/* Add assets for custom JS test mode */
 		if ( $helper->isContentReviewTestModeEnabled() || self::userCanEditJsPage() ) {
 			\Wikia::addAssetsToOutput( 'content_review_test_mode_js' );
 			\JSMessages::enqueuePackage( 'ContentReviewTestMode', \JSMessages::EXTERNAL );
 		}
 
+		/* Add Content Review Module assets for Monobook  */
+		if ( $skin->getSkinName() !== 'monobook' ||
```

Comment:
Shouldn't it be `$skin->getSkinName() !== 'monobook' && self::userCanEditJsPage()` ?